benchmarks/viscoelastic_bending_beam/viscoelastic_bending_beam.prm

```diff
@@ -138,9 +139,9 @@ subsection Material model
   set Model name = viscoelastic
 
   subsection Viscoelastic
-    set Densities            =  2800,  2800,  2800,  2800,  3300
-    set Viscosities          = 1.e18, 1.e18, 1.e18, 1.e18, 1.e24
-    set Elastic shear moduli = 1.e11, 1.e11, 1.e11, 1.e11, 1.e10
+    set Densities            =  2800,  3300
+    set Viscosities          = 1.e18, 1.e24
+    set Elastic shear moduli = 1.e11, 1.e10
```

Comment:
```
  set Densities            =  2800,  3300
  set Viscosities          = 1.e18, 1.e24
  set Elastic shear moduli = 1.e11, 1.e10
```
The parameter files for 3D models with stress and multiple strain invariants are going to look quite a bit simpler 😄

Users will still need to list the values for "composition" type fields in the same order they are listed in Names of fields, but hopefully that does not lead to confusion. We would likely need to update the cookbooks, benchmarks, and tests to include comments to this effect.